maestro-utils/src/test/kotlin/SocketUtilsTest.kt

```diff
@@ -0,0 +1,75 @@
+import maestro.utils.SocketUtils
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import java.net.Inet4Address
+import java.net.InetAddress
+import java.net.NetworkInterface
+import java.net.ServerSocket
+
+class SocketUtilsTest {
+
+    @Test
+    fun `nextFreePort should return a free port within the specified range`() {
+        val from = 5000
+        val to = 5100
+        val port = SocketUtils.nextFreePort(from, to)
+
+        assertTrue(port in from..to)
+    }
+
+    @Test
+    fun `nextFreePort should throw IllegalStateException if no ports are available`() {
```

Comment:
Can we rewrite this test to first use `nextFreePort` to find two suitable ports to use as `from` and `to` instead of hardcoding it? That way it won't clash with other processes running in the environment.

Reply:
Resolved here https://github.com/mobile-dev-inc/maestro/commit/08c9a16a848a1821de0c3429e86f71b5fa1355b1